lib/vrf.h

```diff
@@ -113,7 +113,7 @@ extern struct vrf *vrf_get(vrf_id_t, const char *);
 extern const char *vrf_id_to_name(vrf_id_t vrf_id);
 extern vrf_id_t vrf_name_to_id(const char *);
 
-#define VRF_GET_ID(V, NAME, USE_JSON)                                          \
+#define VRF_GET_ID(V, NAME, USE_JSON)					\
```

Comment:
ok

Reply:
also, VRF_GET_INSTANCE will not systematically create ( vrf_get()) vrfs. I will replace it with a vrf_lookup.
I reword the commit too.
To say it quick, in daemons, vrf will be created either by zebra events, or by config at startup.
For daemons like ospf, lets say, this will be possible to get a vrf context, before connecting to zebra.

a macro VRF_GET_INSTANCE, will be available for usage, whenever one wants to have vrf context.
this will be the case when interfaces will be configured.
VRF_GET_ID is kept because we have some cases where handling a vrf is useless if vrf_id is not defined.
this will be the case when routing entries are in the routing table.

that commit permits on some cases getting the vrf.